hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -454,6 +454,8 @@ enum Status {
     FEATURE_NOT_ENABLED = 86;
 
     INVALID_SNAPSHOT_ERROR = 87;
+
+    BUCKET_SNAPSHOT_EXISTS = 88;
```

Comment:
Is it not supposed to be same as [SNAPSHOT_EXISTS](https://github.com/apache/ozone/pull/4015/files#diff-3d3591532f36caba16c541dff5537ad381a3d6f562ae96b2d37ceaabfb531a84R264)?